parts/k8s/addons/kubernetesmasteraddons-azure-npm-daemonset.yaml

```diff
@@ -76,7 +76,7 @@ spec:
         beta.kubernetes.io/os: linux
       containers:
         - name: azure-npm
-          image: containernetworking/azure-npm:v1.0.13
+          image: containernetworking/azure-npm:v1.0.18-rc2

```

Comment:
looks like this container image is missing from the VHD base image, could you please add it?

See the way `containernetworking/networkmonitor` is pre-pulled as an example https://github.com/Azure/aks-engine/blob/master/packer/install-dependencies.sh#L176

Reply:
Thanks! I already addressed this in https://github.com/Azure/aks-engine/blob/3e7c7b75542b1400ea647a1eb66bea93c03135c0/packer/install-dependencies.sh#L128